setup-scripts/health-checker.py

```diff
@@ -350,9 +350,9 @@ async def check_portal_size():
     # Initialize variables
     num_files = 0
     max_files = 1500000  # 1.5 mln
-    files_text = "Files"
+    files_text = "Files:"
     for line in siac_renter_output.split("\n"):
-        if contains_string(line, files_text):
+        if line.strip().startswith(files_text):
```

Comment:
Can we take it a step further and match with a regex that checks specifically for `^Files:\w+\d+$`?